src/components/plots/piePlot.ts

```diff
@@ -67,6 +79,82 @@ export module Plots {
 
       return allPlotData;
     }
+
+    public sectorValue(): AccessorScaleBinding<D, number>;
+    public sectorValue(sectorValue: number | _Accessor): Plots.Pie<D>;
```

Comment:
This suggests that maybe `_Accessor` should use a generic, so you can specify that it returns number:
```Typescript
public sectorValue(sectorValue: number | _Accessor<number>): Plots.Pie<D>;
```